db/migrate/20151027130308_create_evidences.rb

```diff
@@ -0,0 +1,10 @@
+class CreateEvidences < ActiveRecord::Migration
+  def change
+    create_table :evidences do |t|
+      t.belongs_to :user, index: true
+      t.belongs_to :requirement, index: true
+      t.datetime :valid_until, null: false
```

Comment:
no, I wanted to inherit it from the requirement so when a requirement changes the evidence doesn't :)

comparable to how when old drivers licenses are valid for 5 years, and from now on, newly issued ones are valid for 10 years, your old expiration date still holds :cop: